### PR TITLE
Ternary

### LINKED ISSUE #1234: Ternary Search in Kotlin
Please assign to me.

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1234

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Ternary_Search in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Ternary_Search.kt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Ternary_Search/Ternary_Search.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import java.util.*
+
+//Function for Ternary Search
+fun ternarySearch(array: IntArray, num: Int, left: Int, right: Int):Int {
```
Comment (jainaman224) [inline Ternary_Search/Ternary_Search.kt]: Add space after `:`.

File: Ternary_Search/Ternary_Search.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import java.util.*
+
+//Function for Ternary Search
```
Comment (jainaman224) [inline Ternary_Search/Ternary_Search.kt]: Space after `//`.

Comment (shubhammantri1) [reply Ternary_Search/Ternary_Search.kt]: Done

File: Ternary_Search/Ternary_Search.kt (hunk 1 above)
Comment (shubhammantri1) [reply Ternary_Search/Ternary_Search.kt]: Done

File: Ternary_Search/Ternary_Search.kt [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    return -1
+}
+
+fun main() {
+    val scanner = Scanner(System.`in`)
+    print("Enter the Size of Array : ")
+    val size = scanner.nextInt()
+    val array = IntArray(size)
+    println("Enter entries in Array")
+    for(i in 0 until size) {
```
Comment (jainaman224) [inline Ternary_Search/Ternary_Search.kt]: And space after `for`.

File: Ternary_Search/Ternary_Search.kt [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        array[i] = scanner.nextInt()
+    }
+    print("Number to be searched in array : ")
+    val num = scanner.nextInt()
+    var resultAt = ternarySearch(array, num, 0, array.size)
+    if(resultAt == -1) {
```
Comment (jainaman224) [inline Ternary_Search/Ternary_Search.kt]: Add space after `if`.

**Comment by jainaman224** [comment]
Squash your commits as well.

**Comment by MastersAbh** [comment]
@shubhammantri1 you still working on it ?
It seems @shubhammantri1 has been inactive for a while now. @rishabhgarg25699 @jainaman224

**Comment by MastersAbh** [comment]
@jainaman224 @rishabhgarg25699 The contributor is inactive.

**Pull request closed without merging**